pkg/mapper/mapper_test.go

```diff
@@ -29,18 +29,23 @@ type mappings []struct {
 	maxAge       time.Duration
 	ageBuckets   uint32
 	bufCap       uint32
+	buckets      []float64
 }
 
 func TestMetricMapperYAML(t *testing.T) {
 	scenarios := []struct {
+		testName  string
 		config    string
 		configBad bool
 		mappings  mappings
 	}{
 		// Empty config.
```

Comment:
I think with the `testName` property we can remove the (now duplicate) comments, but it's also fine if we leave them in.

Reply:
Done.